dev/bots/docs.sh

```diff
@@ -22,6 +22,10 @@ function generate_docs() {
     # `dartdoc_options.yaml` to include newly introduced error and warning types.
     "$PUB" global activate dartdoc 1.0.0
 
+    # Install and activate the snippets tool, which resides in the
+    # assets-for-api-docs repo.
```

Comment:
nit: just provide a link on github to the location?

Reply:
Done.